ui/src/main/kotlin/com/algolia/instantsearch/voice/ui/VoiceAndroidView.kt

```diff
@@ -34,17 +34,13 @@ class VoiceAndroidView(
         view.suggestions.text = html
     }
 
-    override fun setTitle(title: VoiceUI.Title) {
-        view.voiceInput.title.setText(title.resource)
-    }
+    override fun setTitle(title: VoiceUI.Title) = view.voiceInput.title.setText(title.resource)
 
     override fun setSubtitle(subtitle: String) {
         view.voiceInput.subtitle.text = subtitle
     }
 
-    override fun setSubtitle(subtitle: VoiceUI.Subtitle) {
-        view.voiceInput.subtitle.setText(subtitle.resource)
-    }
+    override fun setSubtitle(subtitle: VoiceUI.Subtitle) = view.voiceInput.subtitle.setText(subtitle.resource)
```

Comment:
I would be careful here.

If you do this, then the function `setSubtitle` will take the return type of the function `setText`, which happens to be `Unit`. A function that has no explicitly defined return value in Kotlin still has a return type, which is `Unit`. So in this case, it is fine, but it would not be if `setText` was returning a `Boolean` or something else than `Unit`.
This is the equivalent of writing: 

```
override fun setSubtitle(subtitle: VoiceUI.Subtitle) {
        return view.voiceInput.subtitle.setText(subtitle.resource)
}
```

In conclusion I think this pattern should be reserved for getters, and not setters.